app/hotels/src/allHotels/AllHotelsSearchList.js

```diff
@@ -42,47 +106,91 @@ export class AllHotelsSearchList extends React.Component<Props> {
       );
     } else {
       return (
-        <ScrollView style={{ paddingVertical: 3 }}>
-          {hotels.map(edge => {
-            if (edge) {
-              const { node: hotel } = edge;
-              return (
-                <AllHotelsSearchRow
-                  data={hotel}
-                  openSingleHotel={this.props.openSingleHotel}
-                  key={hotel && hotel.id}
-                />
-              );
-            }
-          })}
-        </ScrollView>
+        <View style={styles.container}>
+          <FlatList
+            data={hotels}
+            renderItem={this.renderListItem}
+            keyExtractor={this.keyExtractor}
+            onEndReached={this.loadMore}
+          />
+          {this.renderLoadingIndication()}
```

Comment:
I will have to stop accepting this pattern. I think we have components so let's use them... 🙂